README.md

```diff
@@ -44,25 +44,34 @@ For more information about the project please join our [Discord](https://discord
 ## Git branches and development
 The default branch in the Github repo is `development`. However, releases are cut from the `main` branch. In general, PRs should be made against the `development` branch and reviewed by at least one other person before being merged. When ready for a release, a PR should be made from `development` to `main` and reviewed. Once happy with the PR, it can be merged and then a [release can be drafted for distribution](#automatic-github-distribution).
 
-## Install
+## Contributing
 
-Make sure you are runing node 15.
+You can fork the repo and develop your changes in a new branch there. Try naming it like `feature/awesome-feature-name` or `bug/succinct-name-here` to make things easier. Once you're done with your changes, you can trigger a Pull Request to the main repo, on the `development` branch and someone will take a look at it.
+
+### **Installation**
+
+Make sure you are runing node 16. You can use [nvm](https://github.com/nvm-sh/nvm) to manage your node versions. Run ```yarn install``` in the root directory and it's once done, you can do the same in `src/electron` directory.
 
 ```bash
-yarn
+yarn install
```

Comment:
we can maybe add both commands here:
```suggestion
yarn install && cd src/electron && yarn install
```

Reply:
I don't mind this change as it is more explicit. However here is some info just for awareness. So make your call, either is fine by me.

1. > Running yarn with no command will run yarn install, passing through any provided flags.
source: https://classic.yarnpkg.com/en/docs/cli/#toc-default-command

2. installation of native packages (packages specified in `src/electron/package.json` will happen as a result of the `postinstall` command which is specified in scripts in `package.json`

output of `yarn`
```
$ node -r @babel/register ./config/scripts/CheckNativeDep.js && electron-builder install-app-deps && yarn cross-env NODE_ENV=development webpack --config ./config/webpack/webpack.config.renderer.dev.dll.babel.js && yarn-deduplicate yarn.lock
  • electron-builder  version=22.14.5
  • loaded configuration  file=package.json ("build" field)
  • installing production dependencies  platform=darwin arch=x64 appDir=/Users/dariamikhailova/Dev/tokel/dapp/src/electron
```

Reply:
had no idea of this postinstall script. @rannyeribaptist lets adjust instead for the instruction to be just running `yarn install` once in the root and note in the README that we have a postinstall for the electron dependencies.